### PR TITLE
Adding moonrepo-moon 1.27.6

### PR DESCRIPTION
**Pull request by corker**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by corker** [comment]
@conda-forge/staged-recipes, ready for review!

**Comment by corker** [comment]
@conda-forge/help-rust, ready for review!

**Comment by corker** [comment]
@conda-forge/help-rust, ready for review!

**Comment by corker** [comment]
Hi, @mfansler do you know how long it could take to review my PR? My team has some plans to start using moon with conda forge and it would be great to understand when we could have it.

**Comment by corker** [comment]
@conda-forge-admin, please ping team

**Review by mfansler — CHANGES_REQUESTED** [review]
Thanks for submitting this and apologies for the wait!

I'm not a regular reviewer on the Rust label (be sure to ping @conda-forge/help-rust in the future), but I can at least offer that the recipe needs adjustment to pack the 3rd-party licenses. Otherwise it would look good to me.

**Comment by corker** [comment]
@mfansler is there anything else I need to do to get my PR approved and merged?

**Comment by corker** [comment]
Hi @mfansler it looks like my request is stuck :( What do I need to do to move it forward?

**Review by mfansler — APPROVED** [review]
LGTM

**Comment by mfansler** [comment]
I see now you pinged Rust team correctly - and yes there is a bug that it also pings R team. Unfortunately, the Rust "team" is just one person, so it can be a longer wait to get a review. If you know any Rust developers who would like to help with the workload, please let them know Conda Forge needs volunteers.

File: recipes/moonrepo-moon/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+    - {{ compiler('c') }}              # [not win]
+    - {{ stdlib("c") }}                # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ stdlib("m2w64_c") }}          # [win]
+    - cargo-bundle-licenses
+    - openssl
```
Comment (xhochy) [inline recipes/moonrepo-moon/meta.yaml:23]: This should have been in host

Comment (corker) [reply recipes/moonrepo-moon/meta.yaml:23]: I'm not that good with conda-forge. Can I ask you to explain where should I move this code?

Comment (xhochy) [reply recipes/moonrepo-moon/meta.yaml:23]: This should be in the `host:` section of `requirements`

Comment (corker) [reply recipes/moonrepo-moon/meta.yaml:23]: I see. I double-checked and removed unnecessary deps

**Pull request merged**